yb-voyager/cmd/analyzeSchema.go

```diff
@@ -593,6 +593,72 @@ func checker(sqlInfoArr []sqlInfo, fpath string, objType string) {
 	}
 }
 
+func checkRedundantIndexes() {
+	msr, err := metaDB.GetMigrationStatusRecord()
+	if err != nil {
+		utils.ErrExit("error getting migration status record: %v", err)
+	}
+	if !msr.MigrationAssessmentDone { //TODO: add the case to handle if assessment is run in export schema
```

Comment:
yes. Good point. 
create a function `isMigrationAssessmentDone(msr)` that can abstract out that logic (done directly/done via export-schema).

Reply:
https://yugabyte.atlassian.net/browse/DB-16473